File: client/src/models/House.js
export class House {
  constructor(data) {
    this.id = data.id
    this.year = data.year
    this.price = data.price
    this.imgUrl = data.imgUrl
    this.levels = data.levels
    this.bedrooms = data.bedrooms
    this.bathrooms = data.bathrooms
    this.description = data.description
    this.creatorId = data.creatorId
    this.creator = data.creator
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
  }
}

File: server/Models/House.cs
namespace Gregslist.Models;

public class House
{
  public int Id { get; }
  public string Title { get; set; }
  public string Description { get; set; }
  public int? Year { get; set; }
  public int? Price { get; set; }
  public int? Bedrooms { get; set; }
  public int? Bathrooms { get; set; }
  public string ImgUrl { get; set; }
  public DateTime CreatedAt { get; }
  public DateTime UpdatedAt { get; }
}